src/status_im/translations/zh_wuu.cljs

```diff
@@ -2,239 +2,373 @@
+   :enter-valid-public-key                "请输入有效额公钥或者扫描QR码"
+   :contact-already-added                 "联系人已被添加好叻"
+   :can-not-add-yourself                  "勿能添加侬自家额"
+   :unknown-address                       "伐晓得额地址"
 
-   ;login
+   ;;login
    :connect                               "连接"
    :address                               "地址"
    :password                              "密码"
-   :wrong-password                        "错误的密码"
    :sign-in-to-status                     "登录Status"
    :sign-in                               "登录"
+   :wrong-password                        "密码伐对额"
+   :enter-password                        "输入密码"
 
-   ;recover
-   :passphrase                            "口令短语"
+   ;;recover
+   :passphrase                            "口令短语passphrase"
    :recover                               "恢复"
-   :twelve-words-in-correct-order         "正确排序12个字"
+   :twelve-words-in-correct-order         "正确排序12只字"
 
-   ;accounts
+   ;;accounts
    :recover-access                        "恢复访问"
    :create-new-account                    "建立新帐户"
 
-   ;wallet-qr-code
+   ;;wallet-qr-code
    :done                                  "完成"
-   :main-wallet                           "主要钱包"
 
-   ;validation
-   :invalid-phone                         "无效的电话号码"
-   :amount                                "金额"
-   ;transactions
+   ;;validation
+   :invalid-phone                         "无效额电话号头"
+   :amount                                "数量"
+
+   ;;transactions
    :confirm                               "确认"
    :transaction                           "交易"
+   :unsigned-transaction-expired          "么签名额交易过期叻"
    :status                                "状态"
-   :recipient                             "收件人"
-   :to                                    "至"
+   :recipient                             "收件宁"
+   :to                                    "到"
    :from                                  "从"
    :data                                  "数据"
-   :got-it                                "得到了"
+   :got-it                                "拿到了"
+   :block                                 "块"
+   :hash                                  "哈希"
+   :gas-limit                             "Gas限制"
+   :gas-price                             "Gas价地"
+   :gas-used                              "用特额Gas"
+   :cost-fee                              "价地/费用"
+   :nonce                                 "Nonce"
+   :confirmations                         "确认"
+   :confirmations-helper-text             "请等待至少12只确认，以确保侬额交易安全完成"
+   :copy-transaction-hash                 "拷贝交易哈希"
+   :open-on-etherscan                     "打开 Etherscan.io"
+   :incoming                              "收入"
+   :outgoing                              "花特额"
+   :pending                               "等待中额"
+   :postponed                             "延迟交易"
+
+   ;;webview
+   :web-view-error                        "要系，错特了"})
```

Comment:
Remove the `})` here as otherwise tests fail. Once you have done that, you should be able to add `zh-wuu` to line 270 of `i18n.cljs` and run `lein test-cljs` to confirm for yourself that you have indeed translated all the existing keys in the English file - thank you!